packages/arb-token-bridge-ui/src/components/TransferPanel/TokenRow.tsx

```diff
@@ -268,10 +299,10 @@ export function TokenRow({
       type="button"
       onClick={onClick}
       style={{ ...style, minHeight: '84px' }}
-      disabled={!tokenIsBridgeable}
+      disabled={!tokenIsBridgeable && !withdrawalModeArbOneOrArbGoerliUSDC}
```

Comment:
Great catch! It was needed because initially USDC was in the L2 native list and so it was disabled
but after the USDC-specific handling was added, this boolean is no longer needed